src/UXClient/Components/LinePlot/LinePlot.ts

```diff
@@ -255,7 +255,7 @@ class LinePlot extends Plot {
                     
                     envelope.enter()
                         .append("path")
-                        .attr("class", "valueElement valueEnvelope")
+                        .attr("class", "tsi-valueElement valueEnvelope")
```

Comment:
hell, lets just knock out tsi-valueEnvelope tsi-valueDot tsi-valueLine as part of this as well, should be a relatively easy rename